command/quota_apply_test.go

```diff
@@ -97,3 +99,49 @@ func TestQuotaApplyCommand_Good_JSON(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Len(t, quotas, 1)
 }
+
+func TestQuotaApplyNetwork(t *testing.T) {
+	t.Parallel()
+	// srv, _, _ := testServer(t, true, nil)
+	// defer srv.Shutdown()
+
+	type example struct {
+		q *api.QuotaSpec
+		e string
+	}
+
+	intPtr := func(x int) *int {
```

Comment:
There is `helper.IntToPtr` .  Also, given that it's only used once, I'd simply have a variable (e.g. `sampleInt := 20` ) and reference it in test struct.